docs/FAQ.md

```diff
@@ -71,39 +71,27 @@ We typically do not assign issues to anyone other than long-time contributors. I
 
-- To be a **Discord/Chat** moderator, have an active presence in our chat and
-  have positive engagements with others, while also learning and practicing how
+- To be a **Discord/Chat** moderator, have an active presence in our chat and have positive engagements with others, while also learning
+  and practicing how
   to deal with potential conflicts that may arise.
-- To be a **Forum** moderator, similar to a chat moderator, have an active
-  presence and engage with other forum posters, supporting others in their
-  learning journey, and even giving feedback when asked. Take a look at
-  [The Subforum Leader Handbook](https://forum.freecodecamp.org/t/the-subforum-leader-handbook/326326)
-  for more information.
-- To be a **GitHub** moderator, help process GitHub issues that are brought up
-  to see if they are valid and (ideally) try to propose solutions for these
-  issues to be picked up by others (or yourself).
-
-Altogether, be respectful to others. We are humans from all around the world. With
-that in mind, please also consider using encouraging or supportive language and
-be mindful of cross-cultural communication.
-
-If you practice the above **consistently for a while** and our fellow moderator
-members recommend you, a staff member will reach out and onboard you to the
-moderators' team. Open source work is voluntary work and our time is limited.
-We acknowledge that this is probably true in your case as well. So we emphasize
-being **consistent** rather than engaging in the community 24/7.
-
-Take a look at our [Moderator Handbook](moderator-handbook.md)
-for a more exhaustive list of other responsibilities and expectations we have
+- To be a **Forum** moderator, similar to a chat moderator, have an active presence and engage with other forum posters, supporting
+  others in their learning journey, and even giving feedback when asked. Take a look at
+  [The Subforum Leader Handbook](https://forum.freecodecamp.org/t/the-subforum-leader-handbook/326326) for more information.
+- To be a **GitHub** moderator, help process GitHub issues that are brought up to see if they are valid and (ideally) try to 
+  propose solutions for these issues to be picked up by others (or yourself).
+
+Altogether, be respectful to others. We are humans from all around the world. With that in mind, please also consider using encouraging or supportive language and be mindful of cross-cultural communication.
+
+If you practice the above **consistently for a while** and our fellow moderator members recommend you, a staff member will reach out and onboard you to the moderators' team. Open source work is voluntary work and our time is limited. We acknowledge that this is probably true in your case as well. So we emphasize being **consistent** rather than engaging in the community 24/7.
+
+Take a look at our [Moderator Handbook](moderator-handbook.md) for a more exhaustive list of other responsibilities and expectations we have
 of our moderators.
```

Comment:
```suggestion
```

Reply:
I have made the same changes in my editor as well to see how it looks. I agree with all of the other suggestions that you have given me and i have committed them but i don't get what i am supposed to do in this one.
Don't u think no changes are required here?

Reply:
Oh, if you apply my previous suggestion, the `of our moderators.` text will repeat in lines 88 and 89. We can delete the one in line 89.

Reply:
I guess in that case everything is up to date.

Reply:
But it still duplicated in line 88 and in 89

Reply:
I have removed it, because there seem to be confusion, and we are in the same page but confused.